app/assets/javascripts/discourse/widgets/embedded-post.js.es6

```diff
@@ -29,6 +29,7 @@ export default createWidget("embedded-post", {
   buildKey: attrs => `embedded-post-${attrs.id}`,
 
   html(attrs, state) {
+    attrs.embeddedPost = true;
```

Comment:
I am adding this `attr` because we want to know in `post-cooked` whether this is an `embedded reply to a post` or `a normal post`.

Reply:
Now thinking about this again, couldn't I just look at the `passed widget` type / class name? Maybe.

Reply:
Would love to get @eviltrout eyes on this 🙇

Reply:
This is an OK way to do it. The only slightly awry thing is you are changing attrs inside a widget that’s not using them. It would be arguably better to extend attrs when passing it to post cooked and include the attribute there so that it’s only available where it’s used... but really this is fine!